src/functions/RepresentableFunction.cpp

```diff
@@ -110,8 +110,11 @@ template <int D> bool RepresentableFunction<D>::outOfBounds(const Coord<D> &r) c
     return false;
 }
 
-template class RepresentableFunction<1>;
-template class RepresentableFunction<2>;
-template class RepresentableFunction<3>;
+template class RepresentableFunction<1, double>;
```

Comment:
here we might want to instantiate also for `float` and `std::complex<float>`. Maybe we should have a macro to do it for us, because we'd then be at 12 lines of boilerplate for almost every class. I might have something in my archives...